crates/matrix-sdk/src/notification_settings/rules.rs

```diff
@@ -402,6 +378,34 @@ impl Rules {
 
         Ok(commands)
     }
+
+    /// Apply a list of command to the managed ruleset.
+    ///
+    /// The commands may silently fail because the ruleset may have changed
+    /// between the time the commands were created and the time it is applied.
+    pub(crate) fn apply_commands(&mut self, commands: &[Command]) {
+        for command in commands {
+            self.apply(command);
+        }
+    }
+
+    /// Apply a command to the managed ruleset.
+    ///
+    /// The command may silently fail because the ruleset may have changed
+    /// between the time the command was created and the time it is applied.
+    pub(crate) fn apply(&mut self, command: &Command) {
```

Comment:
Could this be inlined in `apply_commands` instead, and `apply_commands` renamed to `apply` then?

Reply:
👍